twogplus/views.py

```diff
@@ -60,17 +59,34 @@ def upload_cert():
         )
         return render_template("home.html")
 
-    # TODO If a user with the name already exist we should only update it
     username = vaccine_username if is_vaccinated else test_username
-    user = User(username, is_vaccinated, is_tested)
-    db.session.add(user)
-    db.session.commit()
-
-    # TODO set some cookies or shit
```

Comment:
You deleted the comment that we still need to set a cookie but I don't see that you added this functionality. Did you forget to do this or am I missing something?

Reply:
Yes. I set the cookie in line 63 ;-)

Reply:
oh right, below, must have missed that 🙈